README.md

```diff
@@ -12,26 +12,65 @@ A Python wrapper around the [OSRM API](https://github.com/Project-OSRM/osrm-back
 - Intended to work on python 2.7.x and python 3.
 - Open to suggestions !
 
+----
+
+ * [- Installation](#installation)
+ * [- Requires](#requires)
+    * [- Linux](#Linux)
+    * [- Python packages](#python-packages)
+ * [- Sum up command to install](#sum-up-command-to-install)
+    * [- Ubuntu](#ubuntu)
+ * [- Running the test suite](#running-the-test-suite)
+ * [- Usage](#usage)
+    * [- match](#match)
+    * [- route](#route)
+    * [- table](#table)
+    * [- nearest](#nearest)
+    * [- Accessibility isochrones (based on OSRM _table_ service)](#accessibility-isochrones-based-on-osrm-table-service)
+    * [- Trip](#trip)
+    * [- Using a _Point_ instance to avoid confusion between x/y/latitude/longitude](#using-a-point-instance-to-avoid-confusion-between-xylatitudelongitude-)
+    * [- Easily change the host / profile name to query](#easily-change-the-host--profile-name-to-query)
+      * [- By changing the default url](#by-changing-the-default-url-)
+      * [- Or using a new RequestConfig instance, to switch between various url and use basic authentification](#or-using-a-new-requestconfig-instance-to-switch-between-various-url-and-use-basic-authentification-)
+
 ## Installation
 
 ```
 $ pip install osrm
 ```
 
-## Running the test suite
+## Requires
 
-```
-python setup.py test
-```
+### Linux
 
-## Requires
+- libgdal-dev
+
+### Python packages
 
 - polyline
 - numpy
 - pandas
 - geopandas
 - GDAL
 
+
+## Sum up command to install
+
+### Ubuntu
+
+```
+sudo apt-get install libgdal-dev \
+&& curl -o requirements.txt https://raw.githubusercontent.com/ustroetz/python-osrm/master/requirements.txt \
```

Comment:
I would prefer not to encourage bypassing the pipit requirements as a new version should be soon pushed to fix the current issue. @ustroetz